### ISSUE TITLE
[backend + frontend] Merge datafile together with worldmap.json

### ISSUE THREAD

**Issue by thepassle** [Code-for-All/lockdown#100]
Because the datafiles changed, the colour coding of countries is no longer working, we used to this check:
```js
        if (e.properties.data.lockdowns.length === 0) {
          // no known lockdowns
          value = 'green';
        }
```

But since the data has changed, that's not longer there.

What this means if that if we want to render the color of the countries, we now have to request both worldmap.json, and datafile.json. Since we already request worldmap.json, we should merge datafile.json in there, since we only need to add one key.

I propose the following:

```json
{ 
  "type": "Feature", 
  "properties": {
    "iso2":"AF", 
    "iso3":"AFG", 
    "lockdown_status": "No",
    "NAME": "Afghanistan" 
  }, 
  "geometry": "etcetera"
}
```

This means getting rid of the datafile.json, and merging it with worldmap.json. We can then also get rid of the lockdown services.

@ilyasfoo Do you think this is doable?

**Comment by ilyasfoo** [comment]
Yup, this should be quick. My plan is to have a base Worldmap data, and then append lockdown status, output into WorldMap.json
